### ISSUE TITLE
Add paging for collections

### ISSUE THREAD

**Issue by nodesocket** [bobthecow/genghis#199]
We have over two thousand collections in one database, and loading all the collections nearly causes the browser to lock-up.

**Comment by bobthecow** [comment]
Ha! Fair enough. I'll see what I can do :)

**Comment by bobthecow** [comment]
There's probably a similar problem with lots of databases, as well.

**Comment by nodesocket** [comment]
Would be an awesome bonus to be able to search for collections as well.

**Comment by bobthecow** [comment]
Agreed. I've generally had a pretty manageable number of databases and collections, so I've never had to scratch that particular itch. But I can definitely see where it'd be useful.

**Comment by nodesocket** [comment]
@bobthecow hey Justin. Any progress on paging for collections?

**Comment by bobthecow** [comment]
I haven't had a chance to work on this yet. As a workaround, if you build a version of genghis that doesn't pull collection stats you'll probably mitigate most of the issues (other than searchability, of course).

For PHP, comment out this line:

https://github.com/bobthecow/genghis/blob/master/src/php/Genghis/Models/Collection.php#L259

For Ruby, comment out this line:

https://github.com/bobthecow/genghis/blob/master/src/rb/genghis/models/collection.rb#L114

… everything _should_ still work, other than the collection stats popup, but it should also be a _lot_ faster with that many collections.
